src/content/en/tools/chrome-user-experience-report/getting-started.md

```diff
@@ -0,0 +1,216 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+
+{# wf_updated_on: 2017-10-23 #}
+{# wf_published_on: 2017-10-23 #}
+
+# Chrome User Experience Report {: .page-title }
+
+## Getting started
+
+The Chrome User Experience Report is available to explore on [Google BigQuery](https://cloud.google.com/bigquery/), which is a part of the Google Cloud Platform (GCP). To get started, you'll need a Google account ([sign up](https://accounts.google.com/SignUp)), a Google Cloud project that you will use to access the dataset, and basic knowledge of [SQL](https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax#sql-syntax).
```

Comment:
We want lines to wrap at 80 chars to make readability and future changes in PRs/CLs easier to read